### PR TITLE
fix(t8s-cluster/management-cluster): openstackImageNamePrefix

### PR DESCRIPTION
**Pull request by cwrau**

### CODE REVIEW

File: charts/t8s-cluster/templates/_helpers.tpl [hunk 1]
```diff
@@ -30,5 +30,5 @@ sourceRef:
       {{- $hasGPUFlavor = true -}}
     {{- end -}}
   {{- end -}}
-  {{- $hasGPUFlavor -}}
+  {{- $hasGPUFlavor | ternary true "" -}}
```
Comment (tasches) [inline charts/t8s-cluster/templates/_helpers.tpl:33]: Why is it necessary to add the ternary operator? At first glance, it seems unnecessary as nothing else has changed.

Comment (cwrau) [reply charts/t8s-cluster/templates/_helpers.tpl:33]: `{{ if include "t8s-cluster.hasGPUNodes" ([...]) }}` would also succeed when a `false` is being returned from here, as `"false"`, as a string, as it's non-empty, is a truthy value, therefore we switch the false to `` (empty string), which results in a falsey value

File: charts/t8s-cluster/values.yaml [hunk 2]
```diff
@@ -69,6 +69,4 @@ sshKeyName: null
 
 cni: cilium
 
-imageNameTemplate:
-  computePlane: t8s-engine-2004-kube-{{ .builtin.machineDeployment.version }}
-  controlPlane: t8s-engine-2004-kube-{{ .builtin.controlPlane.version }}
+openstackImageNamePrefix: 2004-kube-
```
[the remark this answers is not in the document]
Comment (marvinWolff) [reply charts/t8s-cluster/values.yaml:72]: ping

[the remark this answers is not in the document]
Comment (cwrau) [reply charts/t8s-cluster/values.yaml]: With the new operator, which is needed by this chart by now anyways, we can drop this, your right 👍

**Review by tasches — APPROVED** [review]

**Pull request merged**